services/builder/serviceCaller.go

```diff
@@ -101,7 +108,12 @@ func callTerraformer(ctx *BuilderContext) error {
 }
 
 // callAnsibler passes config to ansibler to set up VPN
-func callAnsibler(ctx *BuilderContext) error {
+func callAnsibler(ctx *BuilderContext, cboxClient pb.ContextBoxServiceClient) error {
+	ctx.Workflow.Stage = pb.Workflow_ANSIBLER
```

Comment:
I would like to see a bit more granular approach with the `description`, i.e. when we are building the VPN, we can set it as `Setting up the VPN`. If we are installing dependencies, again, why not save it as a `description` like `Installing additional dependecies`? Not only for Ansibler, but anywhere there are multiple stages.